.github/actions/setup-python-env/action.yml

```diff
@@ -7,10 +7,10 @@ runs:
         sudo apt update
         sudo apt install libpq-dev gdal-bin graphviz graphviz-dev -y
       shell: bash
-    - name: Set up Python 3.8
+    - name: Set up Python 3.10
       uses: actions/setup-python@v4
       with:
-        python-version: 3.8
+        python-version: 3.10.10
```

Comment:
Yeah, I tried with 3.10 and it complained that no such version is found and provided a link with all the supported versions. There was 3.10.11 available too so we could use that. I just went with 3.10.10 since that was the version I tested on my machine 😄

Reply:
Yup, it seems that 3.10.11 is the latest of 3.10 👍